dl/src/main/scala/com/intel/analytics/bigdl/utils/LoggerAppender.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to Intel Corporation under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * Intel Corporation licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.utils
+
+import org.apache.log4j._
+
+import scala.collection.JavaConverters._
+
+/**
+ * logger appender, which will output the log to files
+ */
+object LoggerAppender {
+
+  val pattern = "%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n"
+
+  /**
+   * redirect log to `filePath`, whose default value is /tmp/bigdl.log
+   *
+   * @param filePath log file path, default value is /tmp/bigdl.log
+   * @return a new file appender
+   */
+  def fileAppender(filePath: String = "/tmp/bigdl.log"): FileAppender = {
```

Comment:
private? And the name is confusing. Rename it to redirectLogToFile. Also the filePath should not be hard coded